PHP/syntax_test_php.php

```diff
@@ -271,297 +271,129 @@ function foo(?stinrg ...$args) {}
 /** @var Properties: class properties. */
 //  ^ keyword.other.phpdoc
 
-/**
- * @return
```

Comment:
This test is already include at https://github.com/sublimehq/Packages/pull/2109/files#diff-704c2dc0506af29e37902450ea766d79R318.